scripts/vars.py

```diff
@@ -0,0 +1 @@
+AP_NAME = "nrc1"
```

Comment:
Prob want to rename this to something like ```HALOW_INTERFACE``` or somethin. ```AP_NAME``` makes it seem like ```nrc1``` is the name of the access point which it isn't, its the name of the network interface of the HaLow hat